src/python/pants/testinfra/BUILD

```diff
@@ -13,36 +12,18 @@ python_library(
     'src/python/pants/testinfra/engine:engine_test_base',
     'src/python/pants/testinfra/subsystem',
   ],
+  # TODO(???): Once we remove `pants_test:testinfra`, reclaim the wheel name
```

Comment:
You can't. In any language an atfiact namereally needs to map to its contents and not lose the mapping. It allows version resolution to work. If you provide the same contents under multiple package names, the resolver is unaware and can place old==1.0.0 and new==1.2.3 in the same claspath/pythonpath, etc and now you have a 1st in path silent wins rest silently fail situation. This is a very real problem in jvm land, it would be great not to do this ourselves.